index.html

```diff
@@ -39,29 +39,234 @@ <h1 id="my-h1">I’m Crispin<br/> Glad to see<br/> you!</h1>
+                </li>
+            </ul>
+            <div class="second-para">
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <div>
+                <ul class="list2-container">
+                    <li class="lists">html</li>
+                    <li class="lists">css</li>
+                    <li class="lists">javascript</li>
+                </ul>
+            </div><br/>
+            <div>
+                <button id="my-btn">See Project</button>
+            </div>
+        </section><br/><br/>
+
+        <section class="third-section">
+            <img src="images/Snapshoot Portfolio2.png"/>
+            <ul class="section-list">
+                <div>
+                    <h2 class="my-h2">Mult-Post<br/>Stories</h2>
+                </div>
+                <li class="other-list" id="li-1">CANOPY</li>
+                <li>
+                    <img src="images/Counter.png"/>
+                    Back End Dev
+                </li>
+                <li>
+                    <img src="images/Counter.png"/>
+                    2022
+                </li>
+            </ul>
+            <div class="second-para">
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <ul>
+                <li class="lists">html</li>
+                <li class="lists">css</li>
+                <li class="lists">javascript</li>
+            </ul><br/>
+            <div>
+                <a href=""><button id="my-btn">See Project</button></a>
+            </div>
+        </section><br/><br>
+        <section class="fourth-section">
+            <img src="images/Snapshoot Portfolio3.png"/>
+            <div>
+                <h2 class="my-h2">Tonic</h2>
+            </div>
+            <ul class="section-list">
+                <li class="other-list" id="li-1">CANOPY</li>
+                <li>
+                    <img src="images/Counter.png"/>
+                    Back End Dev
+                </li>
+                <li>
+                    <img src="images/Counter.png"/>
+                    2022
+                </li>
+            </ul>
+            <div class="second-para">
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <ul>
+                <li class="lists">html</li>
+                <li class="lists">css</li>
+                <li class="lists">javascript</li>
+            </ul><br/>
+            <div>
+                <a href=""><button id="my-btn">See Project</button></a>
+            </div>
+        </section><br/><br/>
+        <section class="fifth-section">
+            <img src="images/Snapshoot Portfolio4.png"/>
+            <ul class="section-list">
+                <div>
+                    <h2 class="my-h2">Mult-Post<br/>Stories</h2>
+                </div>
```

Comment:
### Please do not create a new pull request when a review is given because the next reviewer will need to see the previous comments. [This is the link to the previous review](https://github.com/Bigizi/Portfolio/pull/2).

The previous reviewer did mention that you remove the `div` inside the `ul` element and replaced,e with an `li`. 

kindly fix this.